nautobot/ipam/utils.py

```diff
@@ -75,19 +75,19 @@ def add_available_vlans(vlan_group, vlans):
     Create fake records for all gaps between used VLANs
     """
     if not vlans:
-        return [{"vid": VLAN_VID_MIN, "available": VLAN_VID_MAX - VLAN_VID_MIN + 1}]
+        return [{"vid": vlan_group.min_vid, "available": vlan_group.min_vid - vlan_group.max_vid + 1}]
```

Comment:
This looks reversed to me:

```suggestion
        return [{"vid": vlan_group.min_vid, "available": vlan_group.max_vid - vlan_group.min_vid + 1}]
```